### PR TITLE
User/le2tang/torso head joint ctrl

### PR DESCRIPTION
**Pull request by le2tang**
Add new class BodyControl with home() and handoff() functions that move torso and head into the home and handoff positions defined in the robot configuration yaml

### CODE REVIEW

File: nightingale_body/src/nightingale_body/body_control.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        rospy.loginfo(f"Manipulation Control: waiting for robot configuration service")
+        rospy.wait_for_service(robot_config_service)
+        lookup_client = rospy.ServiceProxy(
+            robot_config_service, RobotConfigurationLookup
+        )
+        rospy.loginfo(
+            "Nightingale Manipulation Control found robot configuration lookup service server"
+        )
+        try:
+            response = lookup_client("home", RobotConfigurationLookupRequest.HEAD)
```
Comment (FelixKHWong) [inline nightingale_body/src/nightingale_body/body_control.py:73]: Is it necessary to run these functions on init? I'm just scared something goes wrong and breaks the tablet holder. Also is this going to make the torso go high on init?

File: nightingale_utils/configs/robot_configuration_lookup.yaml [hunk 2]
```diff
@@ -29,7 +37,7 @@ joint_configurations: {
             joints: [0.23, 0.23, 0.23],
         },
         right_gripper: {
-            names: ["right_gripper_finger1_joint", "right_gripper_finger2_joint", "right_gripper_finger3_joint" ]
+            names: ["right_gripper_finger1_joint", "right_gripper_finger2_joint", "right_gripper_finger3_joint" ],
```
Comment (FelixKHWong) [inline nightingale_utils/configs/robot_configuration_lookup.yaml:40]: not sure if these should be here but doesn't seem to break anything so we'll keep

**Review by FelixKHWong — CHANGES_REQUESTED** [review]

File: nightingale_body/src/nightingale_body/body_control.py (hunk 1 above)
Comment (le2tang) [reply nightingale_body/src/nightingale_body/body_control.py:73]: No movement occurs on init, just looking up the values for the desired states

File: nightingale_utils/configs/robot_configuration_lookup.yaml (hunk 2 above)
Comment (le2tang) [reply nightingale_utils/configs/robot_configuration_lookup.yaml:40]: They are necessary to run the tests, so it's worth keeping here

**Review by nightingale-project — APPROVED** [review]
nice!

**Review by FelixKHWong — APPROVED** [review]

**Pull request merged**